packages/core/src/dispatcher.ts

```diff
@@ -77,7 +77,7 @@ export default class Dispatcher extends EventEmitter {
       await this._ipfs.pubsub.subscribe(
         this.topic,
         this.handleMessage,
-        {timeout: TESTING ? null : IPFS_GET_TIMEOUT}
+        // {timeout: IPFS_GET_TIMEOUT} // ipfs-core bug causes timeout option to throw
```

Comment:
Can you include the bug id in the comment so we know when we can uncomment this again?